xbrowse_server/matchmaker/utilities.py

```diff
@@ -29,18 +32,21 @@ def get_all_clinical_data_for_family(project_id,family_id,indiv_id):
             A JSON object as per MME spec of a patient
     """
     project = get_object_or_404(Project, project_id=project_id)
+    
+    #get contact information if available
+    seqr_project = project.seqr_project 
+    #Ideally these (the default Sam, and matchmaker@broad) should be set at project creation   
+    seqr_project.is_mme_enabled=True 
```

Comment:
setting seqr_project.* fields to hard-coded values here doesn't seem like the desired behavior.

Reply:
not sure if these should be put into the database either though. These are standard constant settings per host. I don't think they belong in the database and settings is the right spot I think